Apps/Common/src/AspNetConfiguration/Modules/Auth.cs

```diff
@@ -232,8 +231,14 @@ public static void ConfigureAuthorizationServices(IServiceCollection services, I
                             policy.Requirements.Add(new FhirRequirement(FhirResource.Encounter, FhirAccessType.Write));
                         });
 
-                    // API Key Policy
-                    options.AddPolicy(ApiKeyPolicy.Write, policy => policy.Requirements.Add(new ApiKeyRequirement(configuration)));
+                    options.AddPolicy(
+                        ClinicalDocumentsPolicy.Read,
+                        policy =>
+                        {
+                            policy.AuthenticationSchemes.Add(JwtBearerDefaults.AuthenticationScheme);
+                            policy.RequireAuthenticatedUser();
+                            policy.Requirements.Add(new FhirRequirement(FhirResource.ClinicalDocuments, FhirAccessType.Write));
```

Comment:
So we have a ClinicalDocumentsPolicy.Read defined but the requirements we add a FhirAccessType.Write.  Is this correct?

Reply:
Fixed